app/accounts/models.py

```diff
@@ -11,7 +11,7 @@ class UserProfile(models.Model):
     Example of extra information can be google user id, facebook user id, etc.
     """
 
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, unique=True, on_delete=models.CASCADE)
```

Comment:
Is this meant to be a foreign key relationship or a one-to-one relationship? Are users intended to have multiple user profiles?
https://docs.djangoproject.com/en/1.9/topics/db/examples/one_to_one/
http://stackoverflow.com/questions/44109/extending-the-user-model-with-custom-fields-in-django